Fix Shred Settings and Quit in Portable mode

mkhon opened the pull request:
- shred winapp2.ini
- shred Chaff models

Fixes #637

az0 commented:
I tested build 1663 on Windows `0

When winapp2.ini is downloaded, it works great. (Thank you.) However, when winapp2.ini is not downloaded, GUI log shows an errors. This could be resolved by adding an "if exists" condition to appending the `personal_cleaners_dir`.

```
Delete 673B C:\temp\BleachBit-4.0.1.1663-portable\BleachBit-Portable\bleachbit.ini
[WinError 2] The system cannot find the file specified: 'C:\\temp\\BleachBit-4.0.1.1663-portable\\BleachBit-Portable\\cleaners': Command to shred C:\temp\BleachBit-4.0.1.1663-portable\BleachBit-Portable\cleaners
Delete 1.3MB C:\temp\BleachBit-4.0.1.1663-portable\BleachBit-Portable\2600_model.json.bz2
Delete 1.7MB C:\temp\BleachBit-4.0.1.1663-portable\BleachBit-Portable\clinton_content_model.json.bz2
Delete 85kB C:\temp\BleachBit-4.0.1.1663-portable\BleachBit-Portable\clinton_subject_model.json.bz2
```

Also, console log shows this error as seen in https://github.com/bleachbit/bleachbit/issues/881
```
Exception while running operation '_gui': ''NoneType' object has no attribute 'get_iter''
Traceback (most recent call last):
  File "C:\projects\bleachbit\bleachbit\Worker.py", line 353, in run_operations
  File "C:\projects\bleachbit\bleachbit\Worker.py", line 174, in clean_operation
  File "C:\projects\bleachbit\bleachbit\GUI.py", line 918, in update_item_size
AttributeError: 'NoneType' object has no attribute 'get_iter'
```

Also, the portable mode should generate a mostly-empty `bleachbit.ini` file, but it doesn't make the file at all. It seems init_configuration() isn't firing

mkhon commented:
Re: NoneType: this branch did not have the fix for #881 (7cd990b2fa3ef94c7e6db6c1ffd4d9e81eba622b) so re-open is invalid.
Also, the fix for #881 included a fix for proper bleachbit.ini re-creation.

mkhon commented:
@az0 I pushed a new version:
- rebased on master (so that PR #953 is included and AttributeError: 'NoneType' and bleachbit.ini re-creation fixes are in)
- added a check for the presence of cleaners_dir
